src/components/Menu.js

```diff
@@ -4,21 +4,51 @@ import openMenu from "../assets/icons/open_menu_icon.svg"
 import closeMenu from "../assets/icons/close_menu_icon.svg"
 import { Link } from "gatsby"
 import logo from "../assets/logos/fac_logo_black.svg"
+import iconLogo from "../assets/logos/fac_round_logo.png"
+import { breakpoint } from "../styles/utils"
 
 const StickyMenuTriangle = styled.div.attrs({
-  className: "fixed pointer",
+  className: "fixed pointer ph2 pr2-s flex items-center justify-between",
```

Comment:
You can use -m for medium screens, -l for large screens and -ns for a mixture of both, but they Tachyons left out -s to place an emphasis on mobile-first styling. (also I think ph2 accomplishes what you're trying to do anyway)